checkov/secrets/runner.py

```diff
@@ -124,6 +125,11 @@ def run(
             {'name': 'EntropyKeywordCombinator', 'path': f'file://{current_dir}/plugins/entropy_keyword_combinator.py'}
         ]
 
+        if bc_integration.daemon_process:
+            # only happens for 'ParallelizationType.SPAWN'
+            bc_integration.setup_http_manager()
+            bc_integration.set_s3_client()
```

Comment:
This change suddenly caused all of our CI pipelines that use Checkov to fail with:

```
File "/.../site-packages/checkov/common/bridgecrew/platform_integration.py", line 397, in set_s3_client
  raise ValueError("Credentials for client are not set")
```

It worked fine previously. We've been using Checkov "offline" without any integration enabled.

Reply:
Works fine using `3.0.40` and below. `3.1.0` and `3.1.1` get the above error. @gruebel

Reply:
@eliasbrange Are you going to make a new issue for this? If so I'd upvote it, otherwise I'd create one as we are also experiencing an issue with this.

Reply:
I can create one!

Reply:
https://github.com/bridgecrewio/checkov/issues/5770